src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,34 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
+description: Setting up the robot for robocup
+slug: /guides/main/robocup-setup
+authors:
+    - Cameron Murtagh (@CMurtagh-LGTM)
+---
+
+## Networking
+
+1. On the robot with a monitor and keyboard, not via `ssh`.
+2. Rename `system/nugusX/etc/systemd/network/40-wlp58s0-robocup.network`
+   to `system/nugusX/etc/systemd/network/30-wlp58s0.network`
+3. Rename `system/defualt/etc/systemd/30-wlp58s0.network` to `system/default/etc/systemd/40-wlp58s0-robocup.network`
+4. Double check that `Address`, in the file now called `30-wlp58s0.network`, is in the subnet you are connecting to,
+   this is the address of this robot.
+   Usually the subnet mask is `255.255.0.0` meaning that the first two numbers define the subnet.
+   Usually the address of the subnet is `192.168.X.X` or `10.1.X.X`.
+5. Also check that `Gateway` and the first `DNS` are correct in this file, both gateway and dns normally end with
+   `.3.1` and are in the subnet.
```

Comment:
Isn't `.3.1` highly specific to the network you are connecting to? I feel like saying "normally" here is misleading.

DNS is also irrelevant for the robocup network (there is no internet to connect to and all communication on this network is either broadcast/multicast or to a specific IP).